### PR TITLE
feat/SSCS-10559: Adding overrideSchedulingListingFields

### PR DESCRIPTION
**Pull request by HarryH96**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10559

### Change description ###
- Adding new class for overrideSchedulingListingFields - which contains the first field for override duration
- SchedulingAndListingFields class now contians OverrideSchedulingListingFields overrideSchedulingListingFields

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SchedulingAndListingFields.java [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SchedulingAndListingFields {
+    private HearingRoute hearingRoute;
```
Comment (MohammedHMCTS) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SchedulingAndListingFields.java:17]: Has the line endings been changed on this file always use LF as its saying the whole file has changed when it hasn't also id keep the space between the variable looks a bit nicer and cleaner

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/OverrideSchedulingListingFields.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OverrideSchedulingListingFields {
+
+    @JsonProperty("overrideDuration")
+    private int overrideDuration;
```
Comment (MohammedHMCTS) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/OverrideSchedulingListingFields.java:20]: Just double-check this is also LF

**Review by MohammedHMCTS — APPROVED** [review]
Just one comment and looks good to me!

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SchedulingAndListingFields.java (hunk 1 above)
Comment (HarryH96) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SchedulingAndListingFields.java:17]: Hmm good shout - I have just checked and on my local they are LF

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/OverrideSchedulingListingFields.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OverrideSchedulingListingFields {
+
+    @JsonProperty("overrideDuration")
```
Comment (TeaWithLucas) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/OverrideSchedulingListingFields.java:19]: No need for @JsonProperty if it matches the field name

Comment (HarryH96) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/OverrideSchedulingListingFields.java]: didnt realise this - thanks!

**Pull request merged**